fix(trends): Add a mention of the confidence function (#3480)
getsentry/sentry-docs: src/docs/product/performance/trends.mdx

![trends Homepage](trend-lists.png)

Trending transactions are calculated first by filtering out transactions with large [throughput](/product/performance/metrics/#throughput-total-tpm-tps) fluctuations, determining the baselines of the two halves of the duration, and calculating their percentage change. Regressions are any percentage changes greater than zero, while improvements are percentage changes less than zero.
Sentry uses a [t-test](https://academic.oup.com/beheco/article/17/4/688/215960) to compare the before and after periods of the transaction and assign a confidence score.

> This confidence score is unbounded, and by default sentry shows trends with a high confidence score. To view trends regardless of the confidence score, add `confidence():>0` to the search bar.

Reviewer: ```suggestion
This confidence score is unbounded, and by default Sentry shows trends with a high confidence score. To view trends regardless of the confidence score, add `confidence():>0` to the search bar.
```

> This confidence score is unbounded, and by default sentry shows trends with a high confidence score. Though if you want to see trends regardless of the score add `confidence():>0` to the search bar.

Reviewer 2: ```suggestion
This confidence score is unbounded, and by default sentry shows trends with a high confidence score. To view trends regardless of the confidence score, add `confidence():>0` to the search bar.
```
